ImperialCollegeLondon/wsi
# Custom subclass

- Move load_extension_files to appropriate position
- Test
- Fix bug in `river_discharge_order` which the test revealed
- Fix bug in error message

I guess full behaviour would be to write a `custom_classes.py` example file that is loaded? Maybe I should move the subclass over to that?

New
- Add custom class from file
- Add defaults to facilitate this
